src/components/ui/hook-form/timeslots/index.tsx

```diff
@@ -41,6 +42,14 @@ const TimeSlotsBase: FC<TimeSlotsBaseProps> = ({
 
     const error = errors[name] ? <>{errors[name]?.message}</> : '';
 
+    const valueField = useWatch({ control, name });
+
+    useEffect(() => {
+        if (Array.isArray(valueField) && valueField.length > 0) {
+            trigger(name);
+        }
+    }, [name, trigger, valueField]);
```

Comment:
Just curious: why did you change it? In two words, what is the purpose of this changes